eng/config/rulesets/Shipping.ruleset

```diff
@@ -96,60 +29,29 @@
     <Rule Id="RS1022" Action="None" /> <!-- https://github.com/dotnet/roslyn/issues/26420: Enable rule RS1022 for Roslyn.sln -->
     <Rule Id="RS1024" Action="Hidden" /> <!-- Compare symbols correctly -->
   </Rules>
-  <Rules AnalyzerId="Microsoft.Composition.Analyzers" RuleNamespace="Microsoft.Composition.Analyzers">
-    <Rule Id="RS0006" Action="Error" />
```

Comment:
NOTE: RSxxxx rules are not removed/changed in this PR. They have just all been moved under the `Roslyn.Diagnostic.Analyzers` node below.